src/pool/pool_scalable.c

```diff
@@ -438,6 +440,13 @@ static umf_result_t scalable_get_name(void *pool, const char **name) {
     return UMF_RESULT_SUCCESS;
 }
 
+static umf_result_t scalable_trim_memory(void *pool, size_t minBytesToKeep) {
+    (void)pool;           // unused
+    (void)minBytesToKeep; // unused
+
+    return UMF_RESULT_SUCCESS;
```

Comment:
Technically it is success, as this function has no promises, but for me it is ERROR NONSUPPORTED